src/transformStateWithClones.js

```diff
@@ -7,7 +7,36 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateCopy = { ...state };
+  const arr = [];
```

Comment:
What is arr? Create more meaningful variables names

Reply:
Also usage of data type in variable name - is a bad practice. It's a list of state copies, isn't it? stateList will be better here

Reply:
done